src/Providers/Suno/src/Sdk/GenerateV2Request.cs

```diff
@@ -0,0 +1,18 @@
+using System.Text.Json.Serialization;
+
+namespace LangChain.Providers.Suno.Sdk;
+
+public class GenerateV2Request
```

Comment:
maybe change classname from GenerateV2Request to something more meaningful like SunoRequest?

Reply:
will recheck when explore V3 request model, may be it can combined in GenerateRequest or just SunoRequest